notebook/static/notebook/js/notebook.js

```diff
@@ -948,6 +948,18 @@ define(function (require) {
     };
 
 
+    Notebook.prototype.delete_marked_cells = function(indices) {
+        indices = this.get_marked_indices();
+        if(indices.length === 0){
+            return;
+        }
+        this._delete_cells(indices);
+    }
```

Comment:
Semicolon here too, since this is an assignment.  When it becomes an ES6 class, we can get rid of the semicolons on the function definitions.